OpenRA.Mods.Common/Effects/RepairIndicator.cs

```diff
@@ -19,18 +19,17 @@ namespace OpenRA.Mods.Common.Effects
 	class RepairIndicator : IEffect
 	{
 		readonly Actor building;
-		readonly string palettePrefix;
 		readonly Animation anim;
 		readonly RepairableBuilding rb;
+
 		int shownPlayer = 0;
 
-		public RepairIndicator(Actor building, string palettePrefix)
+		public RepairIndicator(Actor building)
 		{
 			this.building = building;
-			this.palettePrefix = palettePrefix;
 
 			rb = building.TraitOrDefault<RepairableBuilding>();
```

Comment:
This should be .Trait so that it throws a useful exception instead of NREing below.